packages/stripe-integration/src/stripe-upe/create-stripe-upe-customer-strategy.ts

```diff
@@ -0,0 +1,20 @@
+import { getScriptLoader } from '@bigcommerce/script-loader';
+
+import {
+    CustomerStrategyFactory,
+    toResolvableModule,
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+import StripeUPECustomerStrategy from './stripe-upe-customer-strategy';
+import StripeUPEScriptLoader from './stripe-upe-script-loader';
+
+const createStripeUPECustomerStrategy: CustomerStrategyFactory<StripeUPECustomerStrategy> = (
+    paymentIntegrationService,
+) => {
+    return new StripeUPECustomerStrategy(
+        paymentIntegrationService,
+        new StripeUPEScriptLoader(getScriptLoader()),
+    );
+};
+
+export default toResolvableModule(createStripeUPECustomerStrategy, [{ id: 'stripeupe' }]);
```

Comment:
I see difference between customer and payment strategy. Should `stripeupe` be passed as `id` or as `gateway`?

https://github.com/bigcommerce/checkout-sdk-js/pull/2193/files#diff-e82d9aba9eaef2d31adf89f728c17e3ef625590948a6365d801f763867e25857R20

Reply:
I am not sure, but the way it is now passed (cusomer strategy - id, payment strategy - gateway), is the way it is working. When I set to `id` or `gateway` in both create strategy files, then it does't work.